GitUI/CommandsDialogs/BrowseDialog/GitStatusMonitor.cs

```diff
@@ -469,10 +473,7 @@ private void Update()
                                     }
                                 }
 
-                                if (!cancelToken.IsCancellationRequested)
```

Comment:
Cancellation is done in ScheduleNextInteractiveTime(), then _commandIsRunning is set to false so a new command can be started.
This could allow starting a second command, not wanted.

Reply:
Have a look at line 249, please.

> This could allow starting a second command, not wanted.

There cannot be a command running here (it has finished or been cancelled).
What do you mean with "second command"? How could it be triggered?

Reply:
See line 180, called from FormBrowse.


The Git command runs async, a second invocation can be done in the timer loop.
It occasionally occurred in 3.3 or so when git-status was slow, then git-status running in parallel used all resources.

Reply:
Then rather a1d7b2069102fd0d74940c9f279dce09ea75fefa and let the cancellation do the job, i.e. let it reset `_commandIsRunning`.

Reply:
It may take longer time to start the next refresh (not starting the new until the first is cancelled), but no big difference and the code is easier to understand.

Reply:
I have finally got how it was meant. So mainly renaming and some complements.